libs/core/src/components/sage-image/test/sage-image.e2e.ts

```diff
@@ -0,0 +1,11 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('sage-image', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<sage-image></sage-image>');
+
+    const element = await page.find('sage-image');
+    expect(element).toHaveClass('hydrated');
+  });
+});
```

Comment:
I'd like to see a few more test here. These test would be beneficial as they would render the actual markup we would see in the browser.